migrator/migrator.go

```diff
@@ -462,7 +462,7 @@ func buildConstraint(constraint *schema.Constraint) (sql string, results []inter
 	return
 }
 
-func (m Migrator) GuessConstraintAndTable(stmt *gorm.Statement, name string) (_ *schema.Constraint, _ *schema.Check, table string) {
```

Comment:
Revert this line.

Declare the return value name is better for caller to know that means of each var.

Reply:
@huacnlee i has reverted this line.